src/views/CountryRiskWatch/RiskTable/index.tsx

```diff
@@ -0,0 +1,257 @@
+        },
+        [strings],
+    );
+
+    const displacementRiskData = useMemo(
+        () => listToMap(
+            riskData?.idmc?.map(getDataWithTruthyHazardType).filter(isDefined) ?? [],
+            (data) => data.hazard_type,
+        ),
+        [riskData],
+    );
+
+    const exposureRiskData = useMemo(
+        () => ({
+            ...listToMap(
+                riskData?.raster_displacement_data?.map(
+                    getDataWithTruthyHazardType,
+                ).filter(isDefined) ?? [],
+                (data) => data.hazard_type,
+            ),
+            FI: fiData,
+        }),
+        [riskData, fiData],
+    );
+
+    const informSeasonalRiskData = useMemo(
+        () => ({
+            ...listToMap(
+                riskData?.inform_seasonal?.map(getDataWithTruthyHazardType).filter(isDefined) ?? [],
+                (data) => data.hazard_type,
+            ),
+            WF: wfData,
+        }),
+        [wfData, riskData],
+    );
+
+    const riskTableColumns = useMemo(
+        () => ([
+            createStringColumn<HazardTypeOption, string>(
+                'hazard_type',
+                strings.riskTableHazardTypeTitle,
+                (item) => item.hazard_type_display,
+            ),
+            createStringColumn<HazardTypeOption, string>(
+                'riskScore',
+                strings.riskTableInformTitle,
+                (option) => riskScoreToCategory(
+                    getDisplacementForSelectedMonths(
+                        selectedMonths,
+                        informSeasonalRiskData[option.hazard_type],
+                        'max',
+                    ),
+                    option.hazard_type,
+                ),
+                {
+                    headerInfoTitle: strings.riskTableInformTitle,
+                    headerInfoDescription: (
+                        <div className={styles.informDescription}>
+                            <div>
+                                {strings.riskTableInformDescriptionP1}
+                            </div>
+                            <div>
+                                {strings.riskTableInformDescriptionP2}
+                            </div>
+                            <div>
+                                {resolveToComponent(
+                                    strings.riskTableInformDescriptionP3,
+                                    {
+                                        moreInfoLink: (
+                                            <Link to="https://drmkc.jrc.ec.europa.eu/inform-index/INFORM-Covid-19/INFORM-Covid-19-Warning-beta-version">
+                                                {strings.riskTableInformDescriptionHereLabel}
+                                            </Link>
+                                        ),
+                                    },
+                                )}
+                            </div>
+                        </div>
```

Comment:
Are we sure this will not have a performance penalty?

Reply:
The table only has 5 fixed rows, I'm not sure how far we need to go to optimize it. Also let's benchmark it.

Reply:
It being a header prop should also not affect cell rendering. so it should be pretty safe.